nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py

```diff
@@ -244,6 +244,19 @@ def process_full_tracks(df_all, thresh, pix_size, interval):
     df_full = add_growth_features.add_early_growth_rate(df_full, interval)
     df_full = add_growth_features.add_late_growth_rate_by_endpoints(df_full)
     df_full = add_growth_features.fit_tracks_to_time_powerlaw(df_full, "volume", interval)
+    
+    # For LRM
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'height', 'height_percentile', pix_size) 
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'density', 'density', pix_size)
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'xy_aspect', 'xy_aspect')
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'SA_vol_ratio', 'SA_vol_ratio')
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'SA_vol_ratio', 'SA_vol_ratio')
+    df_full = add_features.add_feature_at(df_full, "frame_transition", 'neighbor_avg_dxdt_48_volume_whole_colony', 'neighbor_avg_dxdt_48_volume_whole_colony')
+    
+    ft_list = ['height', 'density', 'volume', 'mesh_sa', 'xy_aspect', 'SA_vol_ratio', 'neighbor_avg_dxdt_48_volume_whole_colony']
+    multiplier_list = [pix_size, 1 / pix_size**2, pix_size**3, pix_size**2, 1, 1, 1]
```

Comment:
can you determine the multiplier list programatically by calling `get_plot_labels_for_metric`. That will help avoid bugs in the future. 
```
multiplier_list = [get_plot_labels_for_metric(x)[0] for x in ft_list]
```

Reply:
Done!